### PR TITLE
Document how to use persistent volume in grafana

### PR DESCRIPTION
**Pull request by QuentinBisson**
Closes https://github.com/giantswarm/roadmap/issues/223

### CODE REVIEW

File: helm/prometheus-operator-app/values.yaml [hunk 1]
```diff
@@ -788,6 +788,34 @@ grafana:
     #   replacement: $1
     #   action: replace
 
+  ## Enable persistence using Persistent Volume Claims
+  ## ref: http://kubernetes.io/docs/user-guide/persistent-volumes/
+  ##
+  persistence:
+    type: pvc
+    enabled: false
+    # storageClassName: default
```
Comment (TheoBrigitte) [inline helm/prometheus-operator-app/values.yaml:797]: Feels weird that some setting are commented out and other not. I would say we comment out all default, WDYT?

File: helm/prometheus-operator-app/values.yaml [hunk 2]
```diff
@@ -788,6 +788,34 @@ grafana:
...
+      - kubernetes.io/pvc-protection
+    # selectorLabels: {}
+    # subPath: ""
+    # existingClaim:
+
+    ## If persistence is not enabled, this allows to mount the
+    ## local storage in-memory to improve performance
+    ##
+    inMemory:
+      enabled: false
```
[the remark this answers is not in the document]
Comment (QuentinBisson) [reply helm/prometheus-operator-app/values.yaml:812]: No worriies about all your questions, All I did is to take the values from the grafana app (the default values) so users can see the defaults and check out how to configure it without actually going to the grafana app themselves

Regarding the persistency, you can store dashboards and I guess snapshots, create dashboard in the UI and have them stored in a PVC

File: helm/prometheus-operator-app/values.yaml (hunk 1 above)
Comment (QuentinBisson) [reply helm/prometheus-operator-app/values.yaml:797]: It's all a default so I can just comment everything it you rthink it makes sense

Comment (TheoBrigitte) [reply helm/prometheus-operator-app/values.yaml:797]: Ok, let's comment everything then. But I agree it's good to have visibility for us and customer :)

File: helm/prometheus-operator-app/values.yaml (hunk 2 above)
[the remark this answers is not in the document]
Comment (TheoBrigitte) [reply helm/prometheus-operator-app/values.yaml]: Ok, fine then.

File: helm/prometheus-operator-app/values.yaml (hunk 1 above)
Comment (QuentinBisson) [reply helm/prometheus-operator-app/values.yaml:797]: Everything is commented out. I'm sad upstream disagreed with this though

**Pull request closed without merging**